src/drive/styles/table.styl

```diff
@@ -204,7 +204,7 @@ mime-types('fil-file-', '../assets/icons/')
         max-width  100vw
 
     .fil-content-body
-        [class^='coz-spinner']
+        & > [class^='coz-spinner']
```

Comment:
I don't think the `&` selector is useful here ?

Reply:
Indeed. It's optional. I'll remove it.